src/components/AssetCard/AssetCard.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { Props } from './AssetCard.types'
+import VerticalCard from './VerticalCard'
+import HorizontalCard from './HorizontalCard'
+
+export default class AssetCard extends React.PureComponent<Props> {
+  static defaultProps = {
+    horizontal: false
+  }
+
+  render() {
+    const { horizontal, assetId, thumbnail, title } = this.props
+
+    return horizontal ? (
+      <HorizontalCard assetId={assetId} thumbnail={thumbnail} title={title} />
+    ) : (
+      <VerticalCard assetId={assetId} thumbnail={thumbnail} title={title} />
```

Comment:
Is the `assetId` necessary here? I imagine the AssetCard will be called like this `<AssetCard assetId={2} />` (using a container to get the full asset from the state) or maybe `<AssetCard asset={asset} />` but these inner components might be better of receiving a full `asset` or each property they use individually.

How do you imagine this API should look like?

Reply:
Maybe pass an AssetResource?